atc/db/build_preparation.go

```diff
@@ -15,6 +15,7 @@ type MissingInputReasons map[string]string
 const (
 	NoVersionsSatisfiedPassedConstraints string = "no versions satisfy passed constraints"
 	NoVersionsAvailable                  string = "no versions available"
+	NoResourceCheckFinished              string = "no resource check finished after build triggered"
```

Comment:
Should we change this to 'checking for latest available versions'?

Right now it kind of reads like an error state, but in practice we expect this to happen briefly for every manually triggered build, so it might be best to just make it look like a pending/loading/waiting condition and let the user decide if they've been waiting too long. wdyt?

```suggestion
	NoResourceCheckFinished              string = "checking for latest available versions"
```

Reply:
(asked in person :+1: - will merge my own suggestion)